benchmarks/benchmark_matmul.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import sparse
+
+
+class Matmul_Sparse:
+    params = (["coo", "gcxs"], [0.01, 0.33, 0.5, 1.0])
+
+    def setup(self, p, dens_arg):
+        np.random.seed(0)
+        self.x = sparse.random((100, 100), density=0.01, format=p)
+        self.y = sparse.random((100, 100), density=dens_arg, format=p)
```

Comment:
This still produces a GCXS when the dense argument is `1.0`, what I meant in https://github.com/pydata/sparse/pull/467/files#r633073348 was that `0` means `self.x` should be `.todense()`, and `1` means the same for `self.y`.